src/app/components/Persist/ChoosePasswordFields.tsx

```diff
@@ -69,7 +72,6 @@ export function ChoosePasswordFields() {
           >
             <CheckBox
               name="profileStorageUndependableAcknowledge"
-              required
```

Comment:
I would remove this prop in `ChoosePasswordInputFields` password1 field too so validation is consistent.

Reply:
as long as it works everywhere. looks like it's used in more than one place.

Reply:
:+1: removed